mapit/models.py

```diff
@@ -333,7 +333,7 @@ class Meta:
         verbose_name_plural = 'geometries'
 
     def __str__(self):
-        return '%s, polygon %d' % (self.area, self.id)
+        return '%s, polygon %d' % (smart_text(self.area), self.id)
```

Comment:
Q: Why didn’t you use `u''` as an easier way to fix this?
A: Django 1.8, still supported, still supports Python 3.2, therefore so do we. `u''` was only re-added in 3.3.